### ISSUE TITLE
ACCESSIBILITY:

### ISSUE THREAD

**Issue by knpausch** [knpausch/overlook#4]
- [x] Create a branch for accessibility.
- [x] Use this branch to make your dashboard as accessible as possible.
- [x] Push this branch up to GH. You can merge the changes into main but do not delete this branch.
- [x] Accessibility audits should be at 90%-100% for the dashboard.
- [x] A user should be able to interact with all functionality of your application by tabbing through it, no use of the trackpad
- [x] ARIA attributes should be utilized for any UI elements that are not understood by the screen reader

_NOTE: This branch should not have a login page so that during the live eval, we can run the Lighthouse audit and check tabbability of your dashboard (without the login page)._
